at_client/lib/src/util/network_util.dart

```diff
@@ -12,3 +12,9 @@ class NetworkUtil {
     return result;
   }
 }
+
+class NetworkConnectivityChecker {
```

Comment:
Why is this here?

Reply:
I have removed this. I will double check on this.